dependencies.go

```diff
@@ -33,7 +33,7 @@ func (f FastlaneRunner) InstallDependencies(opts EnsureDependenciesOpts) error {
 			f.logger.Println()
 
 			if err := cmd.Run(); err != nil {
-				return f.wrapCommandError(cmd, err)
```

Comment:
This method was adding some text as part of the human readable messaging. Are we removing that to go back to the old style?

Reply:
Nevermind. I see how this is working now.